tests/foreman/ui/test_contenthost.py

```diff
@@ -76,6 +79,16 @@ def setUpClass(cls):
             environment=cls.env,
             organization=cls.session_org,
         ).create()
+        if not settings.cdn:
+            # if not using cdn by settings, have to force uploading the
+            # manifest and attach a subscription to activation key
```

Comment:
And, as i said, if force_manifest_upload  is not adding subscription - then maybe it should?) Better fix the bug than workaround it each time.